### ISSUE TITLE
[bitnami/postgresql-ha] Not posible to define archive_command for incremental backup

### ISSUE THREAD

**Issue by Alvaro-Campesino** [bitnami/charts#20028]
### Name and Version

bitnami/postgresql-ha

### What is the problem this feature will solve?

Right now we have replication but we do not have the ability to define an additional incremental backup in a PVC of choice mounted with extravolumes or if possible created as the main PVCS for data are created.

It will be good either  to be able to either:

-  provide to  the `pg_ctl` a  `-c archive_command=' cp % /var/lib/postgresql/backup/${MY_POD_NAME}` 
-  Generate the `archive_command` value in `postgresql.conf` based on an environment variable

### What is the feature you are proposing to solve the problem?

An additional variable should be added to the values.yaml

`.Values.postgresql.archiveCommand` 

If defined it will be added to the ENV variables in the statefulset.

During startup when the file is configured it could be placed in the postgresql.conf

### What alternatives have you considered?

I tried to use configmaps for the postgresql.conf, but it won't work as each node needs to have different `primary_conninfo` value.
At the same time there is no easy way to add this into the code.

I considered using extraArgs, but those are not args to the `pg_ctl`, but to the command you override it with.

I am going to try to  use a initcontainer or a sidecar to modify the postgresql.conf, but it needs to be done before the postgresql starts so it is doing again all the job already done when the file is created by the image.

**Comment by javsalgar** [comment]
Hi,

Probably adding an env var for the archive command could be the simplest choice. That would be editing libpostgresql,sh in the https://github.com/bitnami/containers. Would you like to submit a PR?

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! The team will take a look
